Source/Microsoft.Teams.Apps.DLLookup/ClientApp/src/App.scss

```diff
@@ -7,8 +7,9 @@
 */
 .default-container {
     height: 100vh !important;
-/*        background-color: #F3F2F1;*/
-        background-color: #FFFFFF;
+    /*        background-color: #F3F2F1;*/
```

Comment:
Delete instead of leaving this commented out

Reply:
Removed